source/jsonnet/england-wales/household/blocks/individual/location_one_year_ago.jsonnet

```
@@ -3,13 +3,31 @@ local rules = import 'rules.libsonnet';
 
 local listName = 'household';
 
-local question(title, description, additionalAnswerOptions=[]) = {
+local questionTitle(isProxy) = (
+  if isProxy then {
+    text: 'One year ago, what was <em>{person_name_possessive}</em> usual address?',
+    placeholders: [
+      placeholders.personNamePossessive,
+    ],
+  }
+  else 'One year ago, what was your usual address?'
+);
+
+local questionDescription(isProxy) = (
+  if isProxy then [
+    'If they had no usual address one year ago, select the address where they were staying',
+    'If the <strong>coronavirus</strong> pandemic affected their usual address one year ago, select where you were living <strong>before your circumstances changed</strong>.',
+  ] else [
+    'If you had no usual address one year ago, select the address where you were staying',
```

I can see these were wrong prior to the changes, but these should end with a full stop.
```suggestion
    'If they had no usual address one year ago, select the address where they were staying.',
    'If the <strong>coronavirus</strong> pandemic affected their usual address one year ago, select where you were living <strong>before your circumstances changed</strong>.',
  ] else [
    'If you had no usual address one year ago, select the address where you were staying.',
```